dis_001_significant_earthquakes/Dockerfile

```diff
@@ -19,11 +19,11 @@ RUN mkdir -p /opt/$NAME/data
 WORKDIR /opt/$NAME/
 COPY contents/ .
 COPY time.cron .
-# VOLUME ./data
+VOLUME ./data
 
 # Set user for container security
 USER root
-RUN adduser -D $NAME
+RUN adduser $NAME
```

Comment:
I guess `-D` just the current default values. Not sure why it was there to begin with.